festim/h_transport_problem.py

```diff
@@ -320,7 +320,7 @@ def need_projecting_solute(self, exports):
             bool: True if the solute needs to be projected, False else.
         """
         need_solute = False  # initialises to false
-        for export in exports.exports:
+        for export in exports:
             if isinstance(export, festim.DerivedQuantities):
                 for quantity in export.derived_quantities:
                     if isinstance(quantity, festim.SurfaceFlux):
```

Comment:
@RemDelaporteMathurin here appears a question on this method which is not tested and not used (?)

Reply:
you mean it's not even used in the main branch?

Reply:
Oh I see on line 311 that we have a TODO to remove this "unused" method, so maybe we can take this opportunity to remove it!